app/main.py

```diff
@@ -1,8 +1,52 @@
+def limitation(
+        value: float, min_value: float, max_value: float) -> float:
+    return max(min(value, max_value), min_value)
+
+
 class Car:
-    # write your code here
-    pass
+    def __init__(
+            self, comfort_class: int, clean_mark: int, brand: str) -> None:
```

Comment:
The best approach of placing parameters is to move them on the next **lines** after function name. One parameter - 1 line.
Closing bracket should be on a separate line